pom.xml

```diff
@@ -9,7 +9,7 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <jdk.version>17</jdk.version>
+        <jdk.version>11</jdk.version>
```

Comment:
Not a problem for this particular task, but I really recommend to uplift local java version to 17 in order not to downgrade version of project. Further tasks may not be compiled on java 11